app/charts/map/helpers.ts

```diff
@@ -178,6 +179,17 @@ export const getBBox = (
   }
 };
 
+export function DeckGLOverlay(
+  props: MapboxOverlayProps & {
+    interleaved?: boolean;
+  }
+) {
+  // @ts-ignore
+  const overlay = useControl<MapboxOverlay>(() => new MapboxOverlay(props));
+  overlay.setProps(props);
+
+  return null;
+}
```

Comment:
Can you add a comment describing what is the purpose of MapboxOverlay ? I see fixing overlapping symbols in the issue, but I do not understand how is the overlay solving the problem.

On another note, this commits seems also to contain refactors.

Reply:
I added a small comment, but generally I also don't fully understand how exactly it fixed the problem – I think the problem was in a way we plotted symbols previously that was not 100% correct and this approach was recommended on DeckGL's site, thus "fix" comment.

Reply:
OK thanks for the explanation.